src/animation/AnimationMixer.js

```diff
@@ -654,7 +654,7 @@ Object.assign( AnimationMixer.prototype, {
 
 		remove_empty_map: {
 
-			for ( var _ in bindingByName ) break remove_empty_map;
+			// for ( var _ in bindingByName ) break remove_empty_map;
```

Comment:
Just a little bit explanation.

This for loop could be writtin like:

```javascript
for(var _ in bindingByName){  // Iterate over object properties
    break;                                  // Break loop immediatly... so does nothing
}
remove_empty_map;              // Does nothing too
```

but instead of comment this line, i could totally remove it ! Or maybe it's an obscure voodoo javascript trick that i don't know yet.. ?